### PR TITLE
Orchestration work for adding new fields to the profile page

### PR DESCRIPTION
**Pull request by rushtong**
This requires UI work in this PR: https://github.com/broadinstitute/firecloud-ui/pull/369

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala [hunk 1]
```diff
@@ -52,10 +77,20 @@ object Profile {
...
+      institutionalProgram = mappedKVPs.get("institutionalProgram").get,
+      programLocationCity = mappedKVPs.get("programLocationCity").get,
+      programLocationState = mappedKVPs.get("programLocationState").get,
+      programLocationCountry = mappedKVPs.get("programLocationCountry").get,
       pi = mappedKVPs.get("pi").get,
+      nonProfitStatus = mappedKVPs.get("nonProfitStatus").get,
+      billingAccountName = mappedKVPs.get("billingAccountName") match {
+        case Some(x) => Some(x)
+        case _ => None
+      },
```
Comment (MattPutnam) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala]: Does this `match` do anything?  Looks like it's effectively an identity operation, unless I'm missing something.

**Comment by MattPutnam** [comment]
:+1: Works for me.  Just one question.

**Comment by davidangb** [comment]
This is legitimately failing tests - I have 6 test failures locally.
```
[info] ScalaTest
[info] Run completed in 9 seconds, 291 milliseconds.
[info] Total number of tests run: 151
[info] Suites: completed 15, aborted 0
[info] Tests: succeeded 145, failed 6, canceled 0, ignored 0, pending 0
[info] *** 6 TESTS FAILED ***
[error] Failed: Total 151, Failed 6, Errors 0, Passed 145
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.service.NIHServiceSpec
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 40 s, completed Jan 13, 2016 5:55:10 PM
```

**Comment by rushtong** [comment]
I did have them working earlier today, will fix that.

**Comment by davidangb** [comment]
This should take advantage of the new profile versioning flag introduced in DSDEEPB-2248. In orchestration, change Profile.currentVersion to 2 (Profile.scala, line 72), and in the UI also increment the expected version to 2 (main.cljs, line 152). With those changes, all previously-registered users - like us - should be prompted to update their profiles as soon as this change goes live. Well, the next time they reload the UI.

**Comment by rushtong** [comment]
So that's how the version # works. Updated in both branches.

**Comment by MattPutnam** [comment]
:+1:

**Pull request merged**